keyboards/handwired/tractyl_manuform/5x6_right/rp2040/config.h

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// wiring of each half
+#define MATRIX_COL_PINS \
+    { A15, B3, B4, B5, B6, B7 }
+#define MATRIX_ROW_PINS \
+    { B12, B13, B14, B15, A8, A10 }
+
+#define DIODE_DIRECTION     COL2ROW
+
+// #define USB_VBUS_PIN        B10 // doesn't seem to work for me on one of my controllers... */
```

Comment:
IIRC, a bunch of the RP2040 community controllers support GP19 for VBUS detection.

```suggestion
#define USB_VBUS_PIN  GP19
```
or:
```suggestion
```